src/Starlight/Ephemeris.hs

```diff
@@ -48,14 +48,14 @@ data Ephemeris = Ephemeris
 fromEphemeris :: Ephemeris -> Orbit
 fromEphemeris Ephemeris{ eccentricity, semimajor, longitudeOfAscendingNode, inclination, argumentOfPerifocus, siderealOrbitPeriod, timeOfPeriapsisRelativeToEpoch }
   = Orbit
-    { eccentricity    = realToFrac eccentricity
-    , semimajor       = realToFrac <$> semimajor
+    { eccentricity
+    , semimajor
     , orientation     = orient
-      (realToFrac <$> fromDegrees longitudeOfAscendingNode)
-      (realToFrac <$> fromDegrees inclination)
-      (realToFrac <$> fromDegrees argumentOfPerifocus)
-    , period          = realToFrac <$> siderealOrbitPeriod
-    , timeOfPeriapsis = realToFrac <$> timeOfPeriapsisRelativeToEpoch
```

Comment:
I have to say, I really like `ghc`’s warnings about unnecessary `realToFrac` usage.